root/etc/cont-init.d/50-certbot

```diff
@@ -120,40 +129,34 @@ else
     EMAILPARAM="--register-unsafely-without-email"
 fi
 
-# update plugin names in dns conf inis
-sed -i 's|^certbot_dns_aliyun:||g' /config/dns-conf/aliyun.ini
-sed -i 's|^certbot_dns_domeneshop:||g' /config/dns-conf/domeneshop.ini
-sed -i 's|^certbot_dns_inwx:||g' /config/dns-conf/inwx.ini
-sed -i 's|^certbot_dns_transip:||g' /config/dns-conf/transip.ini
-
```

Comment:
Not removed, just relocated up under where the `dns-conf` files are copied from `/defaults` just to keep them together.